src/plugins/intel_gpu/src/kernel_selector/auto_tuner.h

```diff
@@ -47,6 +47,8 @@ class TuningCache {
 
     bool NeedsSave() const { return needsSave; }
 
+    static TuningCache* get();
```

Comment:
Why do you left TuningCache::{StoreKernel|RemoveKernel|Save|RemoveKernel_v1|RemoveKernel_v2} functions? I think they will never be used and may be safely removed

Reply:
Good catch. Will remove them in a separate PR.